### DISCUSSION TITLE
Improving Content Discovery Recommendation Engine

### DISCUSSION THREAD

**Discussion by krrishsgk** [sct/overseerr#3263] [Feature Requests]
Hi team,

First off, great project. I’ve setup Overseerr on my home server, and it works great. One area I would love to see a change in is with content discovery/ the recommendation engine. Currently the recos are based on what is most popular on major streaming services, and I’ve found them to be quite meaningless for me own tastes (as popular these days is dominated by Marvel shows and movies and similar). I’ve connected my Plex account, so Overseerr is able to see the movies I’ve added to Plex, and maybe even their rating. Displaying the results from a recommendation engine built on my own data should be able to return better results than sorting by popularity for me. Or maybe we can even connect to the API of a website like movielens.

I would love to contribute to add to this feature. Can we discuss this change and can you help me with how I can contribute? It’ll be my first time contributing.

**Comment by byeuji** [comment]
Adding to this, I think it'd be nice to be able to define your own sorts -- like maybe just an interface with a series data points like: "imdb score", "rotten tomatoes critic score", "rotten tomatoes user score", "metacritic score", "box office revenue", "release date", "MyAnimeList score", "AnimeNewsNetwork Bayesian score", etc.,

and then you grant these things weight, which accumulate into a score (and maybe are ignored if no rating is found, maybe a checkbox for this to ignore or not under certain conditions), which can then be named and added to the dropdown, for sorting desc/asc, and enable setting a default sort in the admin interface. 

Then future updates can add different scores as new sites become available, and just add those as a weight to apply with the data source.

**Comment by otaviocv** [comment]
Hi! I was recently introduced to jellyseerr and feel the same thing. I would love to help design these improvements too.

I loved the studio list feature but it seems limited to American studios. The overall recommendation system seems biased toward American content production. I wonder if there is a way to build these systems by providing full customization options (like custom lists, favorite genres, tastebreakers, favorite source locales, favorite directors, and prizes lists like Golden Globe, Oscar, or any other) or alternative default options that can quickly provide customization alternative without any configuration effort.
